src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const rentalPayOfDay = 40;
+  const shortTerm = 3;
+  const shortTermDiscount = 20;
+  const longTerm = 7;
+  const longTermDiscount = 50;
+  const baseRentalCost = days * rentalPayOfDay;
+
+  if (days >= longTerm) {
+    return baseRentalCost - longTermDiscount;
+  }
+
+  if (days >= shortTerm) {
+    return baseRentalCost - shortTermDiscount;
```

Comment:
The current logic applies only one discount, either short-term or long-term, based on the order of the conditions. If the number of days is greater than or equal to the long-term threshold, the long-term discount is applied, and the short-term discount is ignored. Consider revising the logic to ensure that the correct discount is applied based on the number of days. For example, you might want to apply the long-term discount only if the number of days is greater than or equal to the long-term threshold, and the short-term discount otherwise.